pkg/virt-handler/heartbeat/heartbeat.go

```diff
@@ -24,25 +27,30 @@ import (
 const failedSetCPUManagerLabelFmt = "failed to set a cpu manager label on host %s"
 
 type HeartBeat struct {
-	clientset                 k8scli.CoreV1Interface
+	clientSet                 k8scli.NodeInterface
+	client                    kubecli.ShadowNodeInterface
```

Comment:
Please make the variable name to be more distinguishable, like nodeClient and shadowNodeClient

Reply:
Agree